### PR TITLE
Translated Descriptions

### LINKED ISSUE #1: Translate Descriptions
- [x] Short Description
- [x] Description

CheckIO-to-German/Roadmap#39

### PR DESCRIPTION
**Pull request by DahliaSR**
Closes #1

### CODE REVIEW

File: translations/de/task_description.html [hunk 1]
```diff
@@ -1,33 +1,30 @@
...
-    In mathematics, particularly in linear algebra,
-    a skew-symmetric matrix(also known as an antisymmetric or antimetric) is a square matrix <strong>A</strong> whose transpose
-    is also its negative.
-    This means it satisfies the equation
-    <span style="white-space: nowrap">A = &minus;A<sup>T</sup>.</span>
-    If the entry in the i-th row and j-th&nbsp;column is a<sub>ij</sub>, i.e.
-    <span style="white-space: nowrap">A = (a<sub>ij</sub>)</span>
-    then the symmetric condition becomes
-    <span style="white-space: nowrap">a<sub>ij</sub> = &minus;a<sub>ji</sub>.</span>
+  In der Mathematik, insbesondere in der linearen Algebra, ist eine schiefsymmetrische
```
Comment (GiForce) [inline translations/de/task_description.html]: Is it possible to drop the last "in"?
_In der Mathematik, insbesondere der linearen Algebra, ist..._

**Review by GiForce — COMMENTED** [review]
ok

**Review by GiForce — APPROVED** [review]
....Approving process complete.

File: translations/de/task_description.html (hunk 1 above)
Comment (DahliaSR) [reply translations/de/task_description.html]: gute idee...

**Pull request merged**